test/validators/isBefore.test.js

```diff
@@ -0,0 +1,38 @@
+import { test } from '../testFunctions';
+
+describe('isBefore', () => {
+  it('should validate dates against an end date', () => {
```

Comment:
These tests are for the new behaviour, you should keep the current tests as well in order to not introduce breaking changes. See the `(legacy syntax)` tests that I've done for `isAfter`

Reply:
Fixed!